debian/autobake-deb.sh

```diff
@@ -27,8 +27,10 @@ elif [ -d storage/columnstore/columnstore/debian ]
 then
   # ColumnStore is explicitly disabled in the native Debian build, so allow it
   # now when build is triggered by autobake-deb.sh (MariaDB.org) and when the
-  # build is not running on Travis or Gitlab-CI
-  sed '/-DPLUGIN_COLUMNSTORE=NO/d' -i debian/rules
+  # build is not running on Travis or Gitlab-CI.
+  # Include trailing space in sed command to ensure it only matches the build
+  # line and not build flag stanzas.
+  sed '/   -DPLUGIN_COLUMNSTORE=NO/d' -i debian/rules
```

Comment:
Why these is tabs or spaces are added?

Reply:
the space in sed command to ensure it only matches the debian/rules line 105 and not both 105 and 65.